angular/src/components/view.component.ts

```diff
@@ -246,6 +256,16 @@ export class ViewComponent implements OnDestroy, OnInit {
         }
     }
 
+    toggleFieldCount(field: FieldView) {
+        const f = (field as any);
```

Comment:
Do you need to type cast this to `any`? You added `showCount` as a property in `fieldView`.

Reply:
Was following convention here but you're right - it's unnecessary - so I've removed all instances.